scripts/02_deploy_enumerable_nft_contract_template.ts

```diff
@@ -0,0 +1,26 @@
+import { addressFromContractId, stringToHex } from '@alephium/web3'
+import { Deployer, DeployFunction, Network } from '@alephium/cli'
+import { Settings } from '../alephium.config'
+import { EnumerableNFT } from '../artifacts/ts'
+
+const deployEnumerableNFTTemplate: DeployFunction<Settings> = async (
+  deployer: Deployer,
+  network: Network<Settings>
+): Promise<void> => {
+  const initialFields = {
+    collectionId: stringToHex("collection-id"),
```

Comment:
[Nit]: Does it make sense to use an empty `collectionId` to reduce the template contract state size?